CONTRIBUTING.md

```diff
@@ -35,6 +35,30 @@ proposals against the OCaml distribution. (Code changes, but also
 improvement to documentation or implementation comments, which are
 valuable changes on their own.)
 
+## Workflow
+
+All changes to the OCaml distribution need to be processed through the
+GitHub Pull Request (PR) system.  In order to propose a change, a
+contributor thus needs to have a GitHub account, fork the ocaml/ocaml
+repository, create a branch for the proposal on their fork and submit
+it as a Pull Request on the upstream repository.  (If you are not yet
+familiar with GitHub, don't worry, all these steps are actually quite
+easy!)
```

Comment:
Could you mention that it is still possible to send patches through Mantis? Some people have an allergic reaction to github, and I think it would be reasonable to leave the door open for the old workflow of uploading successive patchsets as attachments on a ticket.